editor/utils/url.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { parse, format } from 'url';
+
+/**
+ * Appends arguments to the query string of the url
+ *
+ * @param  {String} url   URL
+ * @param  {Object} args  Query Args
+ *
+ * @return {String}       Updated URL
+ */
+export function addQueryArgs( url, args ) {
+	const parsedUrl = parse( url, true );
+	const query = { ...parsedUrl.query, ...args };
```

Comment:
I'm a bit sensitive to how spread creates a shallow clone, as it could be marginally more performant to mutate `parsedUrl` throughout this function:

```js
export function addQueryArgs( url, args ) {
	const parsedUrl = parse( url, true );
	Object.assign( parsedUrl.query, args );
	delete parsedUrl.search;

	return format( parsedUrl );
}
```